langchain/callbacks/base.py

```diff
@@ -0,0 +1,154 @@
+"""Base callback handler that can be used to handle callbacks from langchain."""
+
+from abc import ABC, abstractmethod
+from typing import Any, Dict, List
+
+from langchain.schema import AgentAction, LLMResult
+
+
+class BaseCallbackHandler(ABC):
+    """Base callback handler that can be used to handle callbacks from langchain."""
+
+    @abstractmethod
```

Comment:
There's a case to be made for these not being abstract methods and instead having a default implementation of `pass` for all methods. Would make writing custom handlers a lot shorter, and I don't see much downside

Reply:
ha had a similar thought. i guess makes it a bit trickier to know if youve implemented all methods on child class? but i agree that it would make it a lot easier. thoughts @agola11 ?

Reply:
yeah I see both sides. Don't have strong opinions

Reply:
i think im gonna make